app/src/main/java/app/revanced/integrations/settingsmenu/ReVancedSettingsFragment.java

```diff
@@ -160,6 +209,25 @@ private Preference findPreferenceOnScreen(CharSequence key) {
 
         return pref;
     }
+    
+    private void initializeList(Context context, SharedPreferences sharedPreferences, ListPreference listPref, SettingsEnum setting) {
```

Comment:
Can you add Javadocs for the methods you added? That way it will be clear what they do, the method does not initialize the lists.

Reply:
As far as I understand, it updates the values for click listener